types/through2/index.d.ts

```diff
@@ -0,0 +1,37 @@
+// Type definitions for through2 v 2.0
```

Comment:
Why it here? You should install it with `npm i @types/throudh2`

Reply:
I did install @types/through2, but I had to change it. So you see the changed file.
The TransformCallback type was outside the namespace. I moved it inside so I can import and use it.

Reply:
Ok. I see